libs/cart/src/actions/cart.actions.ts

```diff
@@ -3,27 +3,28 @@ import { Action } from '@ngrx/store';
 import { DaffCart } from '../models/cart';
 
 export enum DaffCartActionTypes {
-    CartLoadAction = '[DaffCart] Load Action',
-    CartLoadSuccessAction = '[DaffCart] Load Success Action',
-    CartLoadFailureAction = '[DaffCart] Load Failure Action',
-    AddToCartAction = '[DaffCart] Add To Cart Action',
-    AddToCartSuccessAction = '[DaffCart] Add to Cart Success Action',
-    AddToCartFailureAction = '[DaffCart] Add to Cart Failure Action',    
-    CartResetAction = '[DaffCart] Reset Action',
-    CartResetSuccessAction = '[DaffCart] Reset Success Action',
-    CartResetFailureAction = '[DaffCart] Reset Failure Action'
+  CartLoadAction = '[DaffCart] Cart Load Action',
+  CartLoadSuccessAction = '[DaffCart] Cart Load Success Action',
+  CartLoadFailureAction = '[DaffCart] Cart Load Failure Action',
+  CartCreateAction = '[DaffCart] Cart Create Action',
+  CartCreateSuccessAction = '[DaffCart] Cart Create Success Action',
+  CartCreateFailureAction = '[DaffCart] Cart Create Failure Action',
+  AddToCartAction = '[DaffCart] Cart Add To Cart Action',
+  AddToCartSuccessAction = '[DaffCart] Cart Add to Cart Success Action',
+  AddToCartFailureAction = '[DaffCart] Cart Add to Cart Failure Action',
+  CartResetAction = '[DaffCart] Cart Reset Action',
```

Comment:
Will this action trigger a clearCart API call? Either way, I think you're going to need a clear cart action

Reply:
Yeah I repurposed the reset action for the clear call but didn't rename the actions.

Reply:
Do you think it would make your life much more difficult to rename them to "clear"?

Reply:
Nope, will do.